src/js/media-container.ts

```diff
@@ -298,6 +304,7 @@ class MediaContainer extends globalThis.HTMLElement {
               MediaUIAttributes.MEDIA_CHAPTERS_CUES,
               MediaUIAttributes.MEDIA_WIDTH,
               MediaUIAttributes.MEDIA_HEIGHT,
+              MediaUIAttributes.MEDIA_ERROR_MESSAGE,
```

Comment:
yes, `mediaerrorcode` and `mediaerrormessage` on the media-error-dialog and only `mediaerrorcode` on the media-controller. if it's in one attribute we couldn't exclude the message separately from media-controller. 
`mediaerrorcode` feels like it could be useful on media-controller but the message seems noisy and not very useful there.

Reply:
yup agree. good call.